Presentation.Web/Startup.cs

```diff
@@ -26,18 +28,33 @@ public void Configuration(IAppBuilder app)
                     ValidateAudience = false,
                     ValidIssuer = BearerTokenConfig.Issuer,
                     ValidateIssuer = true,
-
                     IssuerSigningKey = BearerTokenConfig.SecurityKey,
                     ValidateIssuerSigningKey = true,
-
                     ValidateLifetime = true,
                 }
             });
 
-            app.UseNinject(); 
+            app.UseNinject();
             app.Use<ApiRequestsLoggingMiddleware>();
             app.Use<DenyUsersWithoutApiAccessMiddleware>();
             app.Use<DenyModificationsThroughApiMiddleware>();
         }
+
+        private static void InitializeHangfire(IAppBuilder app)
+        {
+            // Initializing the Hangfire scheduler
+            GlobalConfiguration.Configuration.UseSqlServerStorage("kitos_HangfireDB");
```

Comment:
Kan denne string sættes i en variabel der så kan bruges i web.config? Eller kan man ikke det?

Reply:
Det er der ikke noget der forhindrer, men indtil videre er der ikke behov for at justere på den løbende afvikling. 

Den kan altid startes manuelt på {rooturl}/hangfire eller via postman med en global admin cookie. Hensigten er at det kører af sig selv uden indblanding fra menneskelige brugere. Vi giver den nok lige et skub på deployment dagen men efter det skal den "klare sig selv".

Reply:
Hov
jeg så at du refererede til hangfire db... det har aktid været på den måde. Det den peger på er "den connection string der hedder " kitos_HangfireDB" .. og DEN overskrives ved hver deployment så ja den er allerede i config ;-) 
![image](https://user-images.githubusercontent.com/13116889/76824985-f2c93d80-6818-11ea-967e-f0192894054a.png)